### PR TITLE
Add documentation.

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?
make it easier to understand.

## Was the API documentation (openapi.json) updated?
n/a

### CODE REVIEW

File: app/services/refresh_metadata_action.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # Look into identityMetadata for compliant ids and use them to fetch
 # descriptive metadata from Symphony.  Put the fetched value in the descMetadata
 class RefreshMetadataAction
+  # @return [NilClass,Object] returns nil if there was no resolvalble metadata id.
```
Comment (justinlittman) [inline app/services/refresh_metadata_action.rb]: I think resolvable is misspelled.

Also, is this saying "NilClass or Object" or "array of NilClass followed by Object"?

Comment (jcoyne) [reply app/services/refresh_metadata_action.rb]: no, it's saying NilClass or Object may be returned.

Comment (justinlittman) [reply app/services/refresh_metadata_action.rb]: Thanks. Still typo on "resolvable".

Comment (jmartin-sul) [inline app/services/refresh_metadata_action.rb]:
```suggestion
  # @return [NilClass,Object] returns nil if there was no resolvable metadata id.
```

Comment (jmartin-sul) [reply app/services/refresh_metadata_action.rb]: took the liberty of merging since this typo was addressed

**Pull request merged**